### ISSUE TITLE
Home Page - Team services section Accessibility  [Severe]

### ISSUE THREAD

**Issue by AbudZohair** [IEEE-AlAzhar/Official-Website#48]
Accessibility should be fixed to be able to select the text of the service and focus on buttons

**Comment by dvmhmdsd** [comment]
The content card itself is linkable and the paragraphs shouldn't be focusable as the SR can read it.
